<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gausseidel</class>
 <widget class="QMainWindow" name="Gausseidel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gausseidel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="valor1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorindep">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorx4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="agrega">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>x1:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorx2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="iniciar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EJECUTAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ingrese los valores de cada ecuacion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="limpiar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>REINICIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>x3:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="valor3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="valor2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorx1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="matrix">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>EQ 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>EQ 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>EQ 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>EQ 4</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>=</string>
     </property>
    </column>
   </widget>
   <widget class="QDoubleSpinBox" name="valorx3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>x2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>x4:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="valor4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Interacciones</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Error aceptado</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Interac">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Erroraceptado">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>-99.999899999999997</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Gausseidel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
